etcdctl/command/cluster_health.go

```diff
@@ -46,9 +46,10 @@ func handleClusterHealth(c *cli.Context) {
 	}
 
 	// do we have a leader?
-	ep, ls0, err := getLeaderStats(tr, client.GetCluster())
+	cl := client.GetCluster()
+	ep, ls0, err := getLeaderStats(tr, cl)
 	if err != nil {
-		fmt.Println("cluster is unhealthy")
+		fmt.Println("failed to get leader stats from cluster urls", cl)
```

Comment:
I agree with you that it cannot determine the cluster's health reliably. And if the client urls is set correctly, there is large probability that the error indicates unhealthy cluster.
I think as long as this message has more context or reasoning, users could understand what it is talking about more.

What about this:

```
fail to connect to the leader through client urls [http://localhost:4001 http://localhost:4001 http://localhost:4001]
cluster is unhealthy
```